tests/hopp/test_dispatch.py

```diff
@@ -332,6 +339,100 @@ def create_test_objective_rule(m):
 
     # TODO: Update the simulate_with_dispatch function for towers and troughs
 
+def test_wave_dispatch():
+    expected_objective = 74403.6
+    dispatch_n_look_ahead = 48
+
+    data = {
+		"lat": 44.6899,
+		"lon": 124.1346,
+		"year": 2010,
+		"tz": -7,
+	}
+	
+    wave_resource_file = Path(__file__).absolute().parent.parent.parent / "resource_files" / "wave" / "Wave_resource_timeseries.csv"
+    site = SiteInfo(data, solar=False, wind=False, wave=True, wave_resource_file=wave_resource_file)
+
+    default_fin_config = {
+	'batt_replacement_schedule_percent': [0],
+	'batt_bank_replacement': [0],
+	'batt_replacement_option': 0,
+	'batt_computed_bank_capacity': 0,
+	'batt_meter_position': 0,
+	'om_fixed': [1],
+	'om_production': [2],
+	'om_capacity': (0,),
+	'om_batt_fixed_cost': 0,
+	'om_batt_variable_cost': [0],
+	'om_batt_capacity_cost': 0,
+	'om_batt_replacement_cost': 0,
+	'om_replacement_cost_escal': 0,
+	'system_use_lifetime_output': 0,
+	'inflation_rate': 2.5,
+	'real_discount_rate': 6.4,
+	'cp_capacity_credit_percent': [0],
+	'degradation': [0],
+	'ppa_price_input': [25],
+	'ppa_escalation': 2.5
+    }
+  
+    mhk_yaml_path = Path(__file__).absolute().parent.parent.parent / "tests" / "hopp" / "inputs" / "wave" / "wave_device.yaml"
```

Comment:
you can simplify with `from hopp import ROOT_DIR` and `ROOT_DIR.parent / "tests" / "hopp" / "inputs" / "wave" / "wave_device.yaml"`